### DISCUSSION TITLE
Old Folder being created on new install

### DISCUSSION THREAD

**Discussion by Chemie99** [ebkr/r2modmanPlus#682] [Q&A]
I wanted to move the folder location for R2modman files.  I uninstalled, deleted the old folder and reinstalled in the new location.  On launch, the old folder is created with the old profile info.
I tried to delete all references in regedit to no avail.
How do I get it to forget about the old folder location?

**Comment by Chemie99** [comment]
Answering (partly) my own question.  Despite uninstall, somewhere the folder location was being kept persistent.  I was able to change the "data folder directory" setting to the new location to prevent the old one being created.  Only comment would be that uninstall should be removing the settings wherever this was being saved as it persisted after full uninstall and deletion of the old folder (and did not appear to be in registry either).

**Comment by ebkr** [comment]
The old one is always re-created because it stores window size information there.
Your mods should not however be downloading to there once changed.
